sklearn/calibration.py

```diff
@@ -90,6 +114,38 @@ class CalibratedClassifierCV(ClassifierMixin,
         .. versionchanged:: 0.22
             ``cv`` default value if None changed from 3-fold to 5-fold.
 
+    n_jobs : int, default=None
+        Number of jobs to run in parallel.
```

Comment:
Please explain here how parallelism happens for this meta estimator. For instance:

"""
Base estimator clones are fitted in parallel across cross-validation iterations.
Therefore parallelism happens only when cv != "prefit".
"""